src/app/base/components/NodeActionMenu/NodeActionMenu.tsx

```diff
@@ -12,6 +12,10 @@ import type { Node } from "app/store/types/node";
 import { NodeActions } from "app/store/types/node";
 import { canOpenActionForm, getNodeActionTitle } from "app/store/utils";
 
+export enum Label {
+  Toggle = "Take action",
```

Comment:
It might be helpful to use a name that reflects the actual label where possible. With the generic `Toggle` you have to switch contexts and open the file to know what this refers to exactly.

```suggestion
  TakeAction = "Take action",
```

Reply:
Good idea!